nexus/api.go

```diff
@@ -176,3 +203,28 @@ func addContextTimeoutToHTTPHeader(ctx context.Context, httpHeader http.Header)
 	httpHeader.Set(HeaderRequestTimeout, time.Until(deadline).String())
 	return httpHeader
 }
+
+type Link struct {
+	Data []byte `json:"data"`
+	Type string `json:"type"`
+}
+
+var linkBase64Encoding = base64.StdEncoding
+
+func EncodeLink(link *Link) (string, error) {
```

Comment:
Add a docstring please (here and below), this is an exported method.

Reply:
Hm, actually, I don't think we need to export these methods.